### DISCUSSION TITLE
Histogram values being capped at 1000

### DISCUSSION THREAD

**Discussion by ejsmith** [open-telemetry/opentelemetry-dotnet#3650] [Q&A]
I am capturing a `double` histogram value for total time spent processing a queue message. Those times are very often measured in minutes. I am using Elastic APM (8.4) to collect and view the metrics. For some reason when I do a chart showing the max values of the histogram they are capped off at 1000. I'm not sure if this is an issue with Elastic APM or something I'm doing wrong with the Otel library. I see there are bucket boundaries for histograms, but I didn't think that would cap the values though. Anyone have any ideas?

**Comment by ejsmith** [comment]
Also, I am sending these metrics to NewRelic and getting the same behavior of the max values being capped at 1000.

**Reply by cijothomas** [reply]
Not sure what you mean by "Max values of the histogram".. Can you clarify further?

**Reply by mic-max** [reply]
What version are you using? Was this always the case or did it start happening without a change on your end?
I suspect it might be the Elastic APM, try configuring a different exporter for your telemetry like the console exporter

**Reply by ejsmith** [reply]
I'm sending to both Elastic APM and NewRelic and when I build histogram charts with the max values of those metrics, every data point in the histogram is 1000.

**Reply by cijothomas** [reply]
OTel .NET does not (yet) populate Min/Max. Are you expecting that NR/Elastic does that calculation on backend and show it?

**Reply by ejsmith** [reply]
Hmm... does that mean only avg would work correctly?

**Reply by cijothomas** [reply]
I think so... because if SDK is not producing Min/Max, nothing can be done by the backend....
